client/test/e2e/patient/record.spec.js

```diff
@@ -0,0 +1,41 @@
+/* jshint expr: true */
+/* global element, by, browser */
+const chai = require('chai');
+const expect = chai.expect;
+
+const FU = require('../shared/FormUtils');
+const components = require('../shared/components');
```

Comment:
Could you clean up these imports and only require what you need in this test suite?